runelite-client/src/main/java/net/runelite/client/plugins/inventoryviewer/InventoryViewerOverlay.java

```diff
@@ -49,7 +49,9 @@
 
 	private final Client client;
 	private final ItemManager itemManager;
+	private final InventoryViewerConfig config;
 	private boolean hidden;
+	private boolean onInventoryTab;
```

Comment:
Move this field to the plugin and add a `@Getter(AccessLevel.PACKAGE)` annotation so from this class we can call `plugin.isOnInventoryTab()`. There's not really any purpose to have the rest of the event handling in the plugin and save the resulting field here instead.